dev/test-studio/locales/index.ts

```diff
@@ -37,14 +37,16 @@ const nbNO = defineLocaleResourceBundle({
     // Used by `fieldGroupsWithI18n` debug schema type
     'field-groups.group-1': '🇳🇴 Gruppe 1',
     'field-groups.group-2': '🇳🇴 Gruppe 2',
+    'text-divider-title': 'Norsk overraskelse 🇳🇴',
   },
 })
 
-const nbNOBStructureOverrides = defineLocaleResourceBundle({
+export const nbNOBStructureOverrides = defineLocaleResourceBundle({
   locale: 'nb-NO',
   namespace: 'structure',
   resources: {
     'default-definition.content-title': 'Innhold 🇳🇴',
+    'text-divider-title': 'Norsk overraskelse 🇳🇴',
```

Comment:
Did you mean "oversettelse"? "Overraskelse" translates to "surprise". Regardless we should keep it this way! 😄

Reply:
nope, I definitely meant surprise 😄 though I wasn't aware that the term translation and surprise were so close together haha feels fitting!